src/spec/integration/product-type-importer.spec.coffee

```diff
@@ -0,0 +1,108 @@
+{expect} = require 'chai'
+Promise = require 'bluebird'
+ProductTypeImporter = require '../../lib/product-type-import'
+{SphereClient} = require 'sphere-node-sdk'
+{ProjectCredentialsConfig} = require 'sphere-node-utils'
+
+errMissingCredentials = 'Missing configuration in env variables'
+
+argv =
+  projectKey: process.env.SPHERE_PROJECT_KEY
```

Comment:
It takes credentials from .sphere config file now. How would you want to pass project-key to tests?